### PR TITLE
Fix CMakeLists.txt for fuzz testing

### PR DESCRIPTION
**Pull request by antonrohr**
These are fixes for the fuzz tests.

### CODE REVIEW

**Review by danielfenner — APPROVED** [review]
lgtm

File: src/OrbitGl/CMakeLists.txt [hunk 1]
```diff
@@ -140,6 +140,8 @@ target_link_libraries(
 
 if(TARGET OpenGL::GLX AND TARGET OpenGL::OpenGL)
   target_link_libraries(OrbitGl PUBLIC OpenGL::GLX)
+elseif(TARGET OpenGL::GL)
+  target_link_libraries(OrbitGl PUBLIC OpenGL::GL)
```
Comment (beckerhe) [inline src/OrbitGl/CMakeLists.txt:144]: I hope that doesn't break anything. I still remember it was a hassle finding a way that works on platforms. Fingers crossed.

Comment (beckerhe) [reply src/OrbitGl/CMakeLists.txt:144]: (Just wanted to mention that, to raise awareness that upcoming opengl-problems might be related with that change.)

If it compiles on the CI, I'm fine with it.

**Review by beckerhe — APPROVED** [review]
Sorry for the delay. LGTM! Thanks Anton!

File: src/OrbitGl/CMakeLists.txt (hunk 1 above)
Comment (antonrohr) [reply src/OrbitGl/CMakeLists.txt:144]: I am somewhat afraid too, but it seems to be working fine so far.  
I think its also somewhat elegant in the way that it only actually links when the target is available. 

I have to admit, I also did not test this on my windows machine yet, but relied on the CI. I will try it on my machine before I merge. (monday morning)

Comment (beckerhe) [reply src/OrbitGl/CMakeLists.txt:144]: SGTM!

**Pull request merged**